### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by codeviking**
Some quick fixes and cleanup:

* Comment out unused API endpoints.
* Kill some dead code in the UI that was confusing.
* Handle spaces / trimmed output in a fashion that doesn't make for a confusing UX.
* Use a semaphore to prevent slow requests from overwriting the current context.

After merging this, I'm going to recreate the repo so that it's not a fork.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import sys
 import hashlib
 
+
```
Comment (jonborchardt) [inline app.py]: extra line intended?

File: static/app.js [hunk 2]
```diff
@@ -323,14 +309,23 @@ class App extends React.Component {
...
+      },
+      body: JSON.stringify(payload)
     })
     .then(response => response.json())
-    .then(data => this.setState({...data, loading: false}))
+    .then(data => {
+      if (this.currentRequestId === currentReqId) {
+        // If the user entered text by typing don't overwrite it, as that feels
+        // weird. If they clicked it overwrite it
+        const output = choice === undefined ? this.state.output : data.output
```
Comment (jonborchardt) [inline static/app.js]: const output = choice ? this.state.output : data.output

File: static/app.js [hunk 3]
```diff
@@ -323,14 +309,23 @@ class App extends React.Component {
...
+        this.setState({...data, output, loading: false})
+      } else {
+        console.log('Discarded');
```
Comment (jonborchardt) [inline static/app.js]: remove log

File: static/app.js [hunk 4]
```diff
@@ -404,52 +395,34 @@ class App extends React.Component {
...
-        </ListItem>
-      )
+    const logit = logits[idx]
+    const prob = formatProbability(probabilities[idx])
+
+    // get rid of CRs
+    const cleanWord = word.replace(/\n/g, "↵")
+
+    return (
+      <ListItem key={`${idx}-${cleanWord}`}>
```
Comment (jonborchardt) [inline static/app.js]: cleanWord should be sufficient for the key

File: static/app.js [hunk 5]
```diff
@@ -404,52 +395,34 @@ class App extends React.Component {
...
+          {' '}
+          <Token>{cleanWord}</Token>
+        </ChoiceItem>
+      </ListItem>
+    )
   })
 
-const goBack = () => {
-  const lastSpace = output.lastIndexOf(" ")
+  const goBack = () => {
```
Comment (jonborchardt) [inline static/app.js]: can you test that goback still works

File: app.py (hunk 1 above)
Comment (codeviking) [reply app.py]: Nope, removing.  Good catch.

File: static/app.js (hunk 2 above)
Comment (codeviking) [reply static/app.js]: I think it should stay, as empty string would trigger this, and I think we'd want to preserve that, right?

File: static/app.js (hunk 3 above)
Comment (codeviking) [reply static/app.js]: 👍

File: static/app.js (hunk 5 above)
Comment (codeviking) [reply static/app.js]: Yes!

File: static/app.js (hunk 4 above)
Comment (codeviking) [reply static/app.js]: 👍

File: static/app.js (hunk 2 above)
Comment (jonborchardt) [reply static/app.js]: ha, i meant 
const output = !choice ? this.state.output : data.output

**Pull request merged**